core/src/main/java/com/linecorp/armeria/client/retry/FibonacciBackoff.java

```diff
@@ -88,4 +88,27 @@ public String toString() {
                           .add("initialDelayMillis", initialDelayMillis)
                           .add("maxDelayMillis", maxDelayMillis).toString();
     }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private long initialDelayMillis;
+        private long maxDelayMillis;
+
+        FibonacciBackoff build() {
+            return new FibonacciBackoff(initialDelayMillis, maxDelayMillis);
```

Comment:
Should we migrate all factory methods where constructors are used into builder methods? e.g.
https://github.com/line/armeria/blob/ccb29fee0fea5a8657fe3ad2de1fe0a19062818c/core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java#L71-L71